external-crates/move/crates/move-vm-transactional-tests/tests/module_publishing/unpublishable_module.mvir

```diff
@@ -0,0 +1,27 @@
+//# publish
+module 0x42.M {
+    public num(): u64 {
+        label b0:
+            return 42;
+    }
+}
+
+//# publish
```

Comment:
Not necessary for this PR, but would be cool to have a VM mode for the test file with `//# init` maybe